### ISSUE TITLE
paru -G returns EXIT_SUCCESS even when it fails

### ISSUE THREAD

**Issue by vimpostor** [Morganamilo/paru#593]
At the moment, if `paru -G` fails, then the exit code is still zero (success). It would be better, if instead it returned a non-zero exit code (like e.g. `yay -G` does).

For example `paru -G asdf-not-existing-package` returns with 0, giving the impression that it was successful, but it would be better if it returned with a non-zero exit code.

This is also not in alignment with `paru -S asdf-not-existing-package`, which already does return error code 1.

**Comment by getchoo** [comment]
I think I found the problem in `getpkgbuilds()`. On line 184 in `src/download.rs`, when there are warnings given from querying the AUR, the return value `ret` - which is set to 0 at the beginning of the function - has a bitwise operation preformed on it with `ret |= ret`. Since in this case, `ret` is still 0 and and the operation is comparing `ret` with itself, `ret` remains 0.

To fix this, the line with the operation would need be changed from `ret |= ret` to just `ret = 1`. I just made this change and after recompiling, it seems to work:
```
$ ./target/release/paru -G not-existing-package
:: Querying AUR...
:: packages not in the AUR: not-existing-package

$ echo $status
1
